src/Components/App/App.css

```diff
@@ -0,0 +1,9 @@
+.wrap {
+	display: block;
+	margin: auto;
+	width: 300px;
+};
+
+.fonts {
+	font-family: Helvetica, Arial, sans-serif; 
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_react_project/pullRequest?prid=5893380)